samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_eu.properties

```diff
@@ -796,3 +796,6 @@ tag_multitag_singlequestion=Aukera hau hautatuz, galdera hau bezalakoa diren gal
 tag_multitag_singlequestion_all=Sistema konfiguratu dago etiketa hauek kopiatzeko berdina diren galdera orori (agian beste azterketa bati kopiatuko dira, edo beste biltzaile bati).
 
 question_use_rubric=Errubrika batekin ebaluatu dezakezu galdera hau
+
+# S2U-21 Safe Exam Browser errors
+seb_exception_publish_upload=An error occured when trying to add the assessment URL to the Safe Exam Browser configuration
```

Comment:
@pablosanroman Could you help us here?

Reply:
Hello @mpellicer 
seb_exception_publish_upload=Errore bat gertatu da azterketaren URLa gehitu denean nabigatzaile seguruko konfiguraziora.

Reply:
Thanks Pablo!